training_config/seq2seq/bart_cnn_dm.jsonnet

```diff
@@ -0,0 +1,52 @@
+{
+    "train_data_path": "",
+    "validation_data_path": "",
+    "dataset_reader": {
+        "type": "allennlp_models.seq2seq.Seq2SeqDatasetReader",
+        "source_tokenizer": {
+            "type": "pretrained_transformer",
+            "model_name": "bart-large",
+            "add_special_tokens": false
+        },
+        "source_token_indexers": {
+            "tokens": {
+                "type": "pretrained_transformer",
+                "model_name": "bart-large",
+                "namespace": "tokens"
+            }
+        },
+
+        "start_symbol": "<s>",
+        "end_symbol": "</s>",
```

Comment:
Is that needed? Isn't that given by the transformer, i.e., we could get the right values from the huggingface tokenizer?

Reply:
As discussed, this is something we can only get form the PretrainedTransformerTokenzier, but in the Seq2SeqReader we only have access to the AllenNLP tokenizer which doesn't have start/end symbol properties (yet).